analyzers/PapasSim.py

```diff
@@ -169,8 +177,8 @@ def process(self, event):
         #create history node
         #note eventually history will be created by the simulator and passed in
         # as an argument and this will no longer be needed
-        uniqueids = list(event.tracks.keys()) + list(event.ecal_clusters.keys()) + list(event.hcal_clusters.keys())
-        history =  dict( (idt, Node(idt)) for idt in uniqueids )                    
+        uniqueids = sorted(list(event.tracks.keys()) + list(event.ecal_clusters.keys()) + list(event.hcal_clusters.keys()))
+        history = dict((idt, Node(idt)) for idt in uniqueids)
        
         #Now merge the simulated clusters and tracks as a separate pre-stage (prior to new reconstruction)        
         # and set the event to point to the merged cluster
```

Comment:
can we remove the commented code at the end (if uncommented...)? not sure what it does.